config/session.php

```diff
@@ -123,12 +125,9 @@
     | new session cookie is created by the framework for every driver.
     |
     */
-
-    'cookie' => env(
-        'SESSION_COOKIE',
-        str_slug(env('APP_NAME', 'winter'), '_').'_session'
-    ),
-
+    
+    'cookie' => env('SESSION_COOKIE', str_slug(env('APP_NAME', 'winter'), '_') . '_session'),
```

Comment:
Looks like this one came across properly, although it would be nice if it didn't eat the formatting. Easy fix or no?

Reply:
You'd have to embed custom formatting into the printer, which, ehh, we can have a look but no promises 😂